src/components/CompanySelector.vue

```diff
@@ -0,0 +1,226 @@
+<template>
```

Comment:
I like this component, but it is a little bit complex. Have you tried using the [official insantsearch](https://www.algolia.com/doc/guides/building-search-ui/getting-started/vue/) component from Algolia?

They have tons of examples and great out-of-the-box functionality.
Maybe it is something to keep an eye for a future iteration.

Reply:
> I like this component, but it is a little bit complex 
I'm planning a refactor for it. It will affect 90-130 rows on the script. 

> Have you tried using the official insantsearch component from Algolia?
> They have tons of examples and great out-of-the-box functionality.
> Maybe it is something to keep an eye for a future iteration.
No, but I'll check it out. I'll take this as a note for a future iteration. I generally followed [this document](https://firebase.google.com/docs/firestore/solutions/search) by Firebase to create my search component on the app.

Reply:
Hey man, I did a refactor for this component. https://github.com/anonymouslyreportingtool/webapp/pull/7/commits/d1f9e0f1571a9cce79016e6d68656afa8dad3d41

What do you think? 🤔

Reply:
I really liked this refactor!
I have a few questions, though.

What do you think on using a `ul` on the `CompanyList.vue` file?
If the company is not found you could wrap the `p` on a `li` tag.
And in the `CompanyListItem.vue` component, the wrapper could be a `li` instead of a `div`.

What do you think?


---

Additionally, I wanted to know if there is a general guideline on using `v-text="text"`, or the moustache (`{{ text }}`)?
I saw both approaches and would prefer any of them over the other unless there is a specific reason.

Reply:
> I really liked this refactor!
> I have a few questions, though.
> 
> What do you think on using a ul on the CompanyList.vue file?
> If the company is not found you could wrap the p on a li tag.
> And in the CompanyListItem.vue component, the wrapper could be a li instead of a div.
> 
> What do you think?

https://github.com/anonymouslyreportingtool/webapp/pull/7/commits/efde4ca69d5c4602fb6694db5bc8cfce5d7e7bc9

> Additionally, I wanted to know if there is a general guideline on using v-text="text", or the moustache ({{ text }})?
> I saw both approaches and would prefer any of them over the other unless there is a specific reason.

Same here. I'll do a search and let you know.